styles/designs/common/pdf/_common-components.scss

```diff
@@ -78,3 +78,26 @@ $VisuallyHidden: (
     )
 );
 
+//Styles specifically for code snippet tables
+$CodeSnippetLineNumbers: (
```

Comment:
for naming components, we use [BEM (block, element, modifier) ](https://getbem.com/naming/) conventions to keep things clearer. so i think that makes these

```
$CodeSnippet__LineNumbers
$CodeSnippet__LineNumbers--Code
```

feel free to check my work if you want, i'm not 100% it wouldn't be `$CodeSnippet__LineNumbers__Code` if the code is another nested block, instead of a modifier on this type of feature

Reply:
refactored to make code consistent with BEM conventions